packages/ui-form-field/src/FormFieldLayout/index.tsx

```diff
@@ -194,8 +189,7 @@ class FormFieldLayout extends Component<FormFieldLayoutProps> {
         ])}
         css={styles?.formFieldLayout}
         style={{ width }}
-        // @ts-expect-error ts-migrate(2339) FIXME: Property '_messagesId' does not exist on type 'For... Remove this comment to see the full error message
-        aria-describedby={this.hasMessages ? this._messagesId : null}
+        aria-describedby={this.hasMessages ? this._messagesId : undefined}
         ref={this.handleRef}
```

Comment:
strangely here the `ref` causes the  `TS2590: Expression produces a union type that is too complex to represent.` error, but just in the IDE. Likely we cannot fix it, its some TS language server issue.

Reply:
Yes, we encountered this issue before with these <ElementType> components. Hopefully some TS update will solve it later or maybe we can type these better later.